package-lock.json

```diff
@@ -1,1646 +1,8 @@
 {
```

Comment:
I don't really know what's going on in here. I think I have an older version of npm installed so it set the lockfile version back. Not sure why the packages changed though. LMK if I should revert the changes to this file or something.